src/App.jsx

```diff
@@ -14,70 +16,52 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const isSelectedGood = selectedGood ? `${selectedGood} is selected` : 'No goods selected';
```

Comment:
I think it would be better to change the name of this variable, as it currently represents the correct title for the list rather than just a boolean value)